content/07.pathogenesis.md

```diff
@@ -102,82 +155,88 @@ The acute phase response to infection results in the heavy damage of the endothe
+This field integrates the "omics" (genomics, transcriptomics, proteomics, metabolomics, etc.) with bioinformatics and other computational strategies.
+These cutting-edge research approaches hold enormous potential for the study of the complexity of biological systems and human diseases [@doi:10.1016/j.gpb.2012.08.004].
 Over the last decade, systems biology approaches have been used widely to study the pathogenesis of diverse types of life-threatening acute and chronic infectious diseases [@doi:10.1038/s41576-020-0212-5].
-Omics-based studies also provided meaningful information regarding host immune responses and surrogate protein markers in several viral, bacterial and protozoan infections [@doi:10.1002/prca.201300074].
-The complex pathogenesis and clinical manifestations of SARS-CoV-2 infection are not understood adequately yet.
-The genome sequence of the pathogen subsequently allowed its phylogenetic characterization and prediction of its protein expression profile, which is crucial for understanding the pathogenesis and virulence of this novel viral infection.
+Omics-based studies have also provided meaningful information regarding host immune responses and surrogate protein markers in several viral, bacterial and protozoan infections [@doi:10.1002/prca.201300074].
+Though the complex pathogenesis and clinical manifestations of SARS-CoV-2 infection are not yet fully understood, “omics” technologies offer the opportunity to discovery-drive analysis of biological changes associated with SARS-CoV-2 infection.
+For example, previous studies suggest that infection by coronaviruses such as SARS-CoV and MERS-CoV and other viruses is associated with the upregulation of ACE2.
+In several preliminary assays and an analysis of previous microarray data, ACE2 expression was reported to be significantly upregulated following infection of human embryonic kidney cells and human airway epithelial cells [@doi:10.1101/2020.02.24.963348].
+They also reported that direct stimulation with inflammatory cytokines such as type I interferons (e.g., IFN&beta;) resulted in the upregulation of ACE2 in human bronchial epithelial cells, with treated groups showing four-fold higher ACE2 expression than control groups at 18 hours post-treatment [@doi:10.1101/2020.02.24.963348].
+G
+Whether SARS-CoV-2 facilitates the positive regulation of its own transmission between host cells is still unclear, the host immune response itself likely plays a key role in mediating infection-associated pathologies.
+A systems-biology approach allows for analyses such as these to identify possible phenotypic and endophenotypic responses to SARS-CoV-2 infection and to develop new hypotheses about how pathogenesis proceeds.
+
+#### Transcriptomics
+
+In addition to the study described above, two other studies have profiled expression following SARS-CoV-2 infection using human cell lines (Figure {@fig:cell-lines-moi}A).
```

Comment:
This section, including the figure, feels like a lot more detailed dive into a couple datasets than the rest of this review. I leave it to your discretion, but I wonder if it would make the most sense to describe the broad findings here and leave the specifics (actual correlation values, etc) to the source literature.

Reply:
I agree we will probably want to make this section a little less detailed! I had some questions for the original author but figured it would be better to handle that in a separate PR. I don't know what you think about that. The changes here should mostly just be cosmetic because I wasn't quite sure yet how to tie it into the rest of the section.

Also, today someone sent me a [medium post](https://elemental.medium.com/a-supercomputer-analyzed-covid-19-and-an-interesting-new-theory-has-emerged-31cb8eba9d63) (whose representation of several things I have issues with, for the record) but it seems to be addressing some new hype topic (bradykinin) that ties quite closely to some of the material here. 

I think we will want to continue revising this section and figure out exactly what these omics studies say about pathogenesis.